.github/guides/STANDARDS.md

```diff
@@ -37,9 +37,9 @@ You can avoid hacky code by using object-oriented methodologies, such as overrid
 
 ### User Interfaces
 
-* All new player-facing user interfaces must use TGUI, unless they are critical user interfaces.
-* All critical user interfaces must be usable with HTML or the interface.dmf, with tgui being *optional* for this UI.
-	* Examples of critical user interfaces are the chat box, the observe button, the stat panel, and the chat input.
+* All newly implemented player-facing user interfaces must use TGUI, unless they are critical user interfaces.
+* All critical user interfaces must be usable with HTML or the interface.dmf, with TGUI being *optional* (or even disallowed) for this UI.
+	* The most prolific example of critical user interfaces are administrative tooling UIs that must be accessible at all times during a server's life. If it's for something fluffy like the Supply Pod menu that one could "live without", it's not exactly critical. However, we really don't want something like the game panel to have a chance of not loading (The game panel is the way that one would delay/restart during a live round). When in doubt, ask.
```

Comment:
a good definition is anything admins NEED to do their jobs, and anything someone debugging an error could NEED, including someone debugging/solving errors in tgui

Reply:
Though I say that, nothing in the old explanation is wrong. chat and observe have backups, the stat panel just uses tgui's api logic (it's not actually React) and the chat input is bypassable with the command bar

Reply:
There should be a call out for interfaces used for recovering from error states, like the alert() box of `disable-cdn`

Reply:
They do indeed have backups- I might incorporate that in somehow.

Reply:
any updates?

Reply:
funnily enough, restarting and delay are also doable via the chatbar